frontend/templates/cucumber/tests/browser/features/happy.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Happy path
+
+  Successful journey through the system and back to the RP
+
+  @mock-api:success
+  Scenario: Toy Happy Path
```

Comment:
This says Toy - should maybe say CriName

Reply:
Updated.